addon/utils/helper-functions.js

```diff
@@ -203,24 +201,3 @@ export function parseUrl(url) {
 
   return [urlPart, params];
 }
-
-// always exclude jshint or jscs files
-export const excludeRegex = /\.(jscs|jshint)$/i;
-
-/**
- * Find files that have been seen by some tree in the application
- * and require them. Always exclude jshint and jscs files
- *
- * @param filePattern
- * @returns {Array}
- */
-export function requireFiles(filePattern) {
-  let filesSeen = Object.keys(requirejs._eak_seen);
-
-  return filesSeen
-    .filter(
-      (moduleName) =>
-        filePattern.test(moduleName) && !excludeRegex.test(moduleName)
-    )
-    .map((moduleName) => require(moduleName, null, null, true));
```

Comment:
This here is problematic (I'm currently running an embroider + vite project that uses FG - had to patch this out in my project)